src/Requests/POST/CreateOrder.php

```diff
@@ -0,0 +1,49 @@
+<?php
+namespace Krokedil\Ledyer\Payments\Requests\POST;
+
+use Krokedil\Ledyer\Payments\Requests\POST;
+use Krokedil\Ledyer\Payments\Requests\Helpers\Order;
+
+/**
+ * Create order request class.
+ *
+ * Acknowledges an order.
+ */
+class CreateOrder extends POST {
+
+	/**
+	 * CreateSession constructor.
+	 *
+	 * @param int    $order_id   The WC order ID.
+	 * @param string $auth_token The Ledyer auth token.
+	 */
+	public function __construct( $order_id, $auth_token ) {
+		$args = get_defined_vars();
+
+		parent::__construct( $args );
+		$this->log_title = 'Create order';
+		$this->endpoint  = "/v1/authorization-tokens/{$auth_token}/order";
+	}
+
+	/**
+	 * Builds the request args for a POST request.
+	 *
+	 * @return array
+	 */
+	public function get_body() {
+		$order = new Order( $this->arguments['order_id'] );
+
+		return array(
+			'country'                 => WC()->customer->get_billing_country(),
+			'currency'                => $order->get_currency(),
+			'customer'                => $order->get_customer(),
+			'locale'                  => str_replace( '_', '-', get_locale() ),
```

Comment:
Is Ledyer Payments available in for example finland and germany? If it is, maybe we should add something to ensure the format for the locale is always the same independent on what the format for the locale is from WordPress. Since some will be returned as only the country, some will have extra things added to them like formal etc.